Consul/ClusterPeering.cs

```diff
@@ -35,6 +36,75 @@ public class ClusterPeeringTokenResponse
         public string PeeringToken { get; set; }
     }
 
+    /// <summary>
+    /// Represents the status of a cluster peering connection in Consul.
+    /// </summary>
+    public class ClusterPeeringStatus
+    {
+        [JsonProperty("ID")]
+        public string ID { get; set; }
+
+        [JsonProperty("Name")]
+        public string Name { get; set; }
+
+        [JsonProperty("State")]
+        public string State { get; set; }
+
+        [JsonProperty("PeerID")]
+        public string PeerID { get; set; }
+
+        [JsonProperty("PeerServerName")]
+        public string PeerServerName { get; set; }
+
+        [JsonProperty("PeerServerAddresses")]
+        public List<string> PeerServerAddresses { get; set; }
+
+        [JsonProperty("StreamStatus")]
+        public StreamStatus StreamStatus { get; set; }
+
+        [JsonProperty("CreateIndex")]
+        public long CreateIndex { get; set; }
+
+        [JsonProperty("ModifyIndex")]
+        public long ModifyIndex { get; set; }
+
+        [JsonProperty("Remote")]
+        public RemoteInfo Remote { get; set; }
+    }
+
+    /// <summary>
+    /// Details the state of the data stream between peered clusters.
+    /// </summary>
+    public class StreamStatus
```

Comment:
https://github.com/hashicorp/consul/blob/72a358cd02533477536ad4bd2b781f520fa7fac6/api/peering.go#L95
```suggestion
    public class PeeringStreamStatus
```

Reply:
Great, thanks for the reference.